javascript/javascript1/week4/Voice-assistant.js

```diff
@@ -0,0 +1,84 @@
+let userName = ''
+let toDo = []
+
+function date() {
+    const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
+    const today = new Date();
+    const todayDate = today.toLocaleDateString("en-US", options)
+    return `today is: ${todayDate}`
```

Comment:
we don't need a function to print today's date. since all are variables inside the process. we are not operating any operation inside it.